code/modules/reagents/reagent_containers/glass.dm

```diff
@@ -273,14 +273,51 @@
 		ITEM_SLOT_LPOCKET, ITEM_SLOT_RPOCKET,\
 		ITEM_SLOT_DEX_STORAGE
 	)
+	var/kicked_over = FALSE
```

Comment:
doc this pls

Reply:
I actually didn't use this anymore and forgot to remove... but it is yeeted now.